docs/src/main/asciidoc/neo4j.adoc

```diff
@@ -403,6 +403,11 @@ So when you access the `/health/ready` endpoint of your application you will hav
 
 This behavior can be disabled by setting the `quarkus.neo4j.health.enabled` property to `false` in your `application.properties`.
 
+=== Driver metrics
+
+If you are using a metrics extension and specify the config property `quarkus.neo4j.pool.metrics-enabled=true`, the Neo4j extension will
```

Comment:
I'm wondering if it would be better to have a global metrics setting rather than something under `pool`. Wouldn't it be more in line with the other extensions?

If we want to add other metrics, will we add more boolean settings?

Reply:
That's a very good point. At first I was planning to add a `quarkus.neo4j.metrics-enabled` property, but then I saw that there already exists `quarkus.neo4j.pool.metrics-enabled` which enables collection of metrics (internally without exporting them). So what I did instead of adding a new property (which would be confusing due to a very similar name), I changed the meaning of the original property, and now, rather than just enable collection of metrics, it also enables exporting them by the Metrics extension.

We could potentially rename the original one to `quarkus.neo4j.metrics-enabled` and let it do both these things at the same time, but it would break any usage of the original property, so I decided to go with the backward compatible approach. 

In the Agroal extension, we do have two different properties
`quarkus.datasource.jdbc.metrics-enabled` and
`quarkus.datasource.DS_NAME.jdbc.enable-metrics` 
There it makes sense to me because here you can enable metrics for each named data source separately, but in Neo4j we only support one connection pool, so it doesn't seem to have much value to have two separate properties for it.

Reply:
I think I would make it a global metric, deprecate the old one (but still consider it if it's true). If you use the `@Deprecated` annotation, the old one won't be included in the config reference doc.

Reply:
But better wait for @michael-simons 's feedback.

Reply:
When I read the PR I expected the meaning of the property to stay the same plus when SmallRye Metrics is available that our pool metrics are exported (Probably because that's the way we role with Spring Boot, which I actually like).

Apart from that we don't have any plans for metrics apart the pool itself in the driver for the time being (Thinking query metrics (which I would like btw)). But if we do this in the future, I would prefer dedicated a dedicated property for it.

Reply:
OK, I suppose we can keep it that way then and wait for user complaints if any.